.gitignore

```diff
@@ -0,0 +1,76 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I'd prefer not to add this .gitignore since it's not generally applicable.

Reply:
removed